contrib/local/02-contracts.sh

```diff
@@ -27,6 +30,16 @@ wasmd tx wasm instantiate "$CODE_ID" "$INIT" --admin="$(wasmd keys show validato
 
 CONTRACT=$(wasmd query wasm list-contract-by-code "$CODE_ID" -o json | jq -r '.contracts[-1]')
 echo "* Contract address: $CONTRACT"
+
+echo "## Create new contract instance with predictable address"
+wasmd tx wasm instantiate "$CODE_ID" "$INIT" --admin="$(wasmd keys show validator -a)" \
+  --from validator --amount="100ustake" --label "local0.1.0" \
+  --salt=$(echo -n "testing" | xxd -ps) --fix-msg \
```

Comment:
I'm very much in favor of hex instead os base64 for such things. Strings are not an option because they cannot hold all values the user wants to use. E.g. if you use hashes or int counters as the salt, you can end of with non-UTF8 data.

Reply:
with the salt as an argument, we can be more flexible. I make hex default in the decoder. Other options are base64 and ascii. This is an existing component that I will use